Change status bar color of enemy units and buildings to a different color

jefetienne opened the pull request:
Screenshot: https://imgur.com/a/9UWF86T

Currently, status bars for both the player and enemies are green, which can be difficult to differentiate and easy to confuse when units get mixed together.

It uses the color biege, which was randomly found because I couldn't find any documentation on how the color numbers were derived, but I actually think it works well as a color. I didn't choose red (10) only because it already indicates low health.

nextghost commented:
Hi,
this is a nice quality of life improvement but it changes the gameplay just enough that it should be optional. Refer to [this commit](https://github.com/signus-game/signus/commit/da2457fb06696963db890b2218b8992bae67e7d3) to see how to add a checkbox to the Features config menu. You can add placeholder files in English to the Czech language directory, I'll translate them myself.

Signus renders the game screen using the 8-bit palette below (defined in `signus-data/data/nolang/graphics/palette.col`):

![palette](https://user-images.githubusercontent.com/1794396/151677938-3563f5c8-7576-44cc-ad27-69e94f075afb.png)

Color 483 is out of range but it works only because the value gets truncated to 0xE3 (227). Colors 0x00 and 0x3F are reserved but you can pick any other color from the palette. I'd also recommend picking new color for low health enemies.

Please follow the new code style: indent by one tab per level, all code blocks wrapped in braces (including one-liners after a loop statement or condition), opening brace on the same line as the preceding statement. You don't need to reformat the whole function, just the few lines which you're changing. And please don't leave any spaces or tabs at the end of the line. `git diff` will highlight trailing whitespace for you.

jefetienne commented:
I have added the option, however I do not know why I am unable to see the text in the checkbox. I have made sure the list was expanded to size 86, did a `make clean` to both the signus and signus-data folders, and even made another clone to my repo with no luck.

I have made the low-health color for enemies be purple. I added the setting under the 'Interface' ini which I think is the correct placement (this will conflict with my video-skipping PR).

nextghost commented:
You also need to run `make install` or copy the modified `texts.dat` to the game data directory. The executable does not read any data files from the build directory.

Using the ternary operator would make the inner conditions more readable:

    clr = (sz < 6) ? 10 /*red*/ : 59 /*green*/;

Color 117 is almost invisible in the health bar. Here's a small color test, colors from left to right:
- 0x3B/0x0A (original)
- 0xE3/0x75 (current PR)
- 0xE6/0x9C
- 0xE6/0x79

![colortest](https://user-images.githubusercontent.com/1794396/151706733-a414c950-5ee6-47d7-b8a2-12ed531ddb12.png)

I think that 0xE6/0x9C looks the best. The low-health color is reddish but still clearly distinct from the player's red.

jefetienne commented:
Updated the colors!

nextghost commented on signus/src/signus.cpp:
```diff
@@ -31,6 +31,7 @@
 #include <time.h>
 #include <climits>
 #include <SDL_timer.h>
+#include <iostream>
```
Here's a bit of leftover debug code. Please delete it so that I can merge.

nextghost commented:
Thanks, squashed and merged.
